servatrice/src/servatrice_database_interface.cpp

```diff
@@ -199,14 +199,18 @@ bool Servatrice_DatabaseInterface::registerUser(const QString &userName,
                                                 const QString &realName,
                                                 ServerInfo_User_Gender const &gender,
                                                 const QString &password,
+                                                bool passwordNeedsHash,
                                                 const QString &emailAddress,
                                                 const QString &country,
                                                 bool active)
 {
     if (!checkSql())
         return false;
 
-    QString passwordSha512 = PasswordHasher::computeHash(password, PasswordHasher::generateRandomSalt());
+    QString passwordSha512 = password;
```

Comment:
Might be more efficient to set this in an if/else?

Reply:
this is simpler, less error prone